MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/SearchGoogleBooks.java

```diff
@@ -0,0 +1,206 @@
+package com.izor066.android.mediatracker.ui;
+
+import android.app.Activity;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.EditText;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.api.services.books.model.Volume;
+import com.google.api.services.books.model.Volumes;
+import com.izor066.android.mediatracker.GoogleBooks.Search;
+import com.izor066.android.mediatracker.MediaTrackerApplication;
+import com.izor066.android.mediatracker.R;
+import com.izor066.android.mediatracker.api.model.Book;
+import com.izor066.android.mediatracker.ui.adapter.SearchResultsAdapter;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class SearchGoogleBooks extends AppCompatActivity implements TextView.OnEditorActionListener, SearchResultsAdapter.OnResultClickListener {
+
+    String TAG = getClass().getSimpleName();
+
+    private String searchString = "";
+    private final String IMAGE_PLACEHOLDER = "https://s.gr-assets.com/assets/nophoto/book/blank-133x176-8b769f39ba6687a82d2eef30bdf46977.jpg";
+    EditText searchGoogleBooks;
+    private SearchTask task;
```

Comment:
This can be local in `onEditorAction`, I cannot seem to find anywhere else it's being used.